LICENSE

```diff
@@ -1,15 +1,178 @@
-Software License Agreement (Apache License)
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
 
-Copyright 2012 Open Source Robotics Foundation
```

Comment:
Do we need to keep the copyright line?

Reply:
I think a `LICENSE` file doesn't typically have a copyright line

* https://github.com/ros2/rclcpp/blob/rolling/LICENSE

Reply:
Got it. Thanks!